src/videojs.ads.js

```diff
@@ -151,10 +151,10 @@ var
       i,
       suppressedTracks = [],
       snapshot = {
-        ended: player.ended(),
-        src: player.currentSrc(),
         currentTime: player.currentTime(),
-        type: player.currentType()
+        ended: player.ended(),
+        source: player.currentSource(),
+        src: player.currentSrc()
```

Comment:
Yes, `src` is used to validate the ad against the current content.

https://github.com/chemoish/videojs-contrib-ads/blob/feature/currentSource/src/videojs.ads.js#L303
https://github.com/chemoish/videojs-contrib-ads/blob/feature/currentSource/src/videojs.ads.js#L308
https://github.com/chemoish/videojs-contrib-ads/blob/feature/currentSource/src/videojs.ads.js#L413
https://github.com/chemoish/videojs-contrib-ads/blob/feature/currentSource/src/videojs.ads.js#L451

`type` on the other hand was only used to regenerate the `src()` call, so I removed it.